modules/aws/sonar-base-instance/setup.tpl

```diff
@@ -77,9 +77,9 @@ function attach_disk() {
 
 function install_tarball() {
     echo Downloading tarball..
-    # Download intallation tarball
+    # Download installation tarball
     TARBALL_FILE=$(basename ${installation_s3_key})
-    /usr/local/bin/aws s3 cp s3://${installation_s3_bucket}/${installation_s3_key} ./$TARBALL_FILE >/dev/null
+    /usr/local/bin/aws s3 cp s3://${installation_s3_bucket}/${installation_s3_key} --region ${installation_s3_region} ./$TARBALL_FILE >/dev/null
```

Comment:
Usually, `src` and `dst` files are adjacent. I guess it works. I would put the region either before `src` or after `dst` like you did with `statistics` module